### DISCUSSION TITLE
Scaffolding app using just Javascript

### DISCUSSION THREAD

**Discussion by XieKaiwen** [vikejs/vike#1622] [Discussion]
I noticed that by scaffolding my app using Bati, the only option available is to scaffold an app with TypeScript which I am not familiar with. 

I tried to configure the scaffolded app to react but it gives me a problem where when i deleted the tsconfig.json, it give me errors "Cannot find type definition file for 'vike-react' The file is in program because entry point of type library 'vike react'" specified in compilerOptions

Is there a way to solve this issue?

**Comment by brillout** [comment]
This question isn't relate to Vike. I think it's best we focus this discussion board around Vike.
